### DISCUSSION TITLE
Busco grupo para  hackaton, tengo algo de experiencia con node js

### DISCUSSION THREAD

**Discussion by willyghost117** [Launch-X-Latam/MisionFrontEnd#532] [Team Posts]
Busco grupo tengo experiencia en node js, soy de Guatemala

**Comment by willyghost117** [comment]
lx09830@innovaccion.mx

**Comment by RoneyVaLo5** [comment]
Hola, tengo un poco de conocimientos en programación en general y lo básico de FrontEnd y BackEnd  si quieres podemos hacer un grupo.

lx07280@innovacion.mx

**Comment by FernandoSic** [comment]
Hola, también soy de Guatemala

Tengo conocimientos básicos de programación, Si gustas podemos formar equipo

lx10826@innovaccion.mx

**Comment by CralpCode** [comment]
Yo también soy de Guatemala y ando buscando grupo tambien, tengo conocimientos en Html ,Css , Javascript y un poco de React, si me aceptan yo uno a su grupo.

lx07376@innovaccion.mx

**Comment by mafer23** [comment]
Metanse Al servidor de discord y hay se buscar para que no hallan spam

**Comment by mafer23** [comment]
https://discord.gg/QZa2UAHu
